canonical-web-and-design/ubuntu.com
# Remove index_de and /c/contact-us

## Done

- I found two odd things to fix
    - Remove index_de and add a redirect to the homepage
    - remove the obviously mistaken - /c/contact-us path

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/index_de
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
